tests/testthat/test_tune_tuneIrace.R

```diff
@@ -11,7 +11,7 @@ test_that("tuneIrace", {
   ctrl = makeTuneControlIrace(maxExperiments = n, nbIterations = 1L, minNbSurvival = 1)
   tr1 = tuneParams(makeLearner("classif.rpart"), multiclass.task, rdesc, par.set = ps1, control = ctrl)
   expect_true(getOptPathLength(tr1$opt.path) >= 10 && getOptPathLength(tr1$opt.path) <= n)
-  expect_true(!is.na(tr1$y))
+  expect_number(tr1$y, lower = 0, upper = 0.2)
```

Comment:
Well, we want to make sure that the optimizer does not run in the wrong direction. This is a quite save range.